### DISCUSSION TITLE
Quetzacoatl: Linux version ?

### DISCUSSION THREAD

**Discussion by Donmanouche** [publicsamples/Quetzalcoatl#10] [Ideas]
Quetzacoatl looks good! Any Linux version foreseen as for some other plugins?
Thank you!

**Comment by Donmanouche** [comment]
I have just seen your answer in the "issues" thread.
I can compile it myself but I would need few explanations to do so. 
Thank you!

**Reply by RickTaylor79** [reply]
Hi, sorry for the late reply! Here's a linux build. If you try it let me know how you get on with it. 

https://github.com/publicsamples/Quetzalcoatl/releases/tag/0.5.3

If you'd like to try building it yourself, try going through these instructions:

https://github.com/publicsamples/Quetzalcoatl/wiki/Build-it-yourself

**Reply by Donmanouche** [reply]
Thank you !
I'll try it and get back to you.
